prometheus-exporters/openstack-exporter/alerts/cinder.alerts

```diff
@@ -101,6 +108,7 @@ groups:
       severity: info
       tier: vmware
       service: cinder
```

Comment:
Potential further improvement: If we define the service in the `values.yaml` we should look it up here to avoid having to maintain it in multiple areas.